revpi-hat-eep/src/lib.rs

```diff
@@ -26,18 +15,18 @@ pub enum RevPiError {
 /// [serde_json](https://docs.serde.rs/serde_json/).
 ///
 /// # Example JSON configuration
-/// ```
+/// ```json
 /// {
 ///     "version": 1,
 ///     "vstr": "KUNBUS GmbH",
-///     "pstr": "RevPi Core3+ 8GB",
-///     "pid": 302,
+///     "pstr": "RevPi ExampleDevice 8GB",
+///     "pid": 666,
```

Comment:
Another one, please see comment at example.json.

Reply:
And still I like them, don't see anything problematic.